gazelle/docs/index.md

```diff
@@ -0,0 +1,45 @@
+# Gazelle Plugin
+
+[Gazelle][gazelle] is a build file generator for Bazel projects. It can
+create new {code}`BUILD` or {code}`BUILD.bazel` files for a project that
```

Comment:
nit: 
```suggestion
create new `BUILD` or `BUILD.bazel` files for a project that
```

I would expect this to be still markdown enough so that the above works.